docs/src/api/class-accessibility.md

```diff
@@ -2,6 +2,8 @@
 * since: v1.8
 * langs: csharp, js, python
 
+**DEPRECATED** This class is deprecated. Please follow our [accessbility testing](../accessibility-testing.md) guide if you need to test page accessibility.
```

Comment:
We have `accessibility-testing.md` only for JS as of today. So either docslint or our docs site will end up being red if this lands.

Reply:
OK, let's hold this until we have the docs and axe integration in all languages.